CairoMakie/src/screen.jl

```diff
@@ -81,8 +81,8 @@ end
 to_cairo_antialias(aa::Int) = aa
 
 """
-* `px_per_unit = 2.0`: see [figure docs](https://docs.makie.org/stable/documentation/figure/).
-* `pt_per_unit = 0.75`: see [figure docs](https://docs.makie.org/stable/documentation/figure/).
+* `px_per_unit = 2.0`: see [figure docs](https://docs.makie.org/stable/documentation/figure_size/).
```

Comment:
@jkrumbiegel I guess this is the old url and should be figure now? Or is this the correct fix?

Reply:
hm that page does exist and in principle contains the right info. But I feel like I have rewritten that section at some point to explain the new changes. Or haven't I? It was too long ago..

Reply:
ah the github interface for some reason didn't show the right changes, ok yeah I see now that I deleted that page in the refactor. So yeah link needs to be fixed